ReactWindows/ReactNative.Shared.Tests/Modules/WebSocket/WebSocketOptionsTests.cs

```diff
@@ -0,0 +1,150 @@
+﻿using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+using ReactNative.Modules.WebSocket;
+
+namespace ReactNative.Tests.Modules.WebSocket
+{
+    [TestFixture]
+    public class WebSocketOptionsTests
+    {
+        [Test]
+        [Category("WebSocket")]
+        public void WebSocketOptions_NoProxy()
+        {
+            var json = @"{
+    origin: 'www.example.com'
+}";
+            var jOptions = JObject.Parse(json);
+            var options = new WebSocketOptions(jOptions);
+            Assert.IsFalse(options.UseDefaultProxy);
```

Comment:
Assert.That(options.UseDefaultProxy, Is.False);
// use the fluent style assertion syntax consistently throughout these new additions, please :)